Caches/test/EteTests/testRunner.py

```diff
@@ -62,81 +59,15 @@ def chechTestResult(goldenPath, testResult):
     print("Executable main file: % s" % args.executable)
     print("Directory containing tests: % s" % args.test_dir)
 
-    testResult = runTest(args.executable, args.test_dir + "test1.dat")
-    checkResult = chechTestResult(args.test_dir + "golden/test1.ans.dat", testResult)
-    print(checkResult)
+    test_files = [f for f in os.listdir(args.test_dir) if f.endswith('.dat') and not f.startswith('golden')]
 
-    testResult = runTest(args.executable, args.test_dir + "test2.dat")
-    checkResult = chechTestResult(args.test_dir + "golden/test2.ans.dat", testResult)
-    print(checkResult)
+    for test_file in test_files:
```

Comment:
test_path and test_files and golden_path  also better to rename using CamelCase.